outlines/structure.md

```diff
@@ -0,0 +1,56 @@
+# Application structure and code style
+
+1. What's different between Meteor and other kinds of apps?
+    1. All JavaScript, enables code sharing
+    2. Structure is flexible
+    3. Currently everything is about magic file names
+    4. App scope vs. package scope vs. file scope variables
+2. Client vs. server vs. both
+3. Smooth path from prototype to production, three sizes of apps, S, M, L
+4. General stuff
+    1. Filenames - named the same as the export
+        1. Every file exports one thing
+    2. Avoid depending on load order via namespacing and Meteor.startup
+        1. If you depend on load order, your best bet is a package
+    3. Naming guidelines for everything
+        1. https://github.com/meteor/guide/issues/9#issuecomment-148800085
+    4. Templates should have a CSS class that matches the template name, and other Blaze suggestions
+    5. Meteor ES2015 style guide (link to it, don't reproduce all of it)
+    6. Perhaps this should include copypasta boilerplate - a standard template/component, a standard publication, standard route, etc? Basically the stuff a scaffolding framework would generate for you?
+5. Small app
+    1. Directory structure around features, not client/server
```

Comment:
> But then, there are always some exceptions. Methods which have to operate on both Posts and Comments and the same time. Publish functions which return multiple collections (like all Posts and Comments tagged with the same tag). Where to put then those?

I think this is totally fine! Posts is just a "theme" to group your code. If there is a publication that returns posts and their associated comments, it seems totally reasonable for that to be under the `Posts` namespace. I don't see why this means that things need to be split up into different modules just because they interact with more than one collection.